portal/db.py

```diff
@@ -61,6 +61,47 @@ def init_db_command():
     click.echo("Initialized the database.")
 
 
+def insert_users():
+    """Insert created users from csv file into database"""
+    # connect to the database
+    con = get_db()
+    cur = con.cursor()
```

Comment:
con and cur should be closed immediately after using them, or use `with` statements